api/config/config_local.json

```diff
@@ -101,7 +101,6 @@
     "enableEventDateModifier": false,
     "enableResearchPurposePrompt": false,
     "enableGenomicExtraction": true,
-    "enableFireCloudV2Billing" : true,
```

Comment:
Feature flag removal should happen in two steps: the config should be left in until the following release, see https://docs.google.com/document/d/133JzAcrIf_ZKFqzL2jvVaBbxb69NMMSjs7iTuj8xW0A/edit#bookmark=id.d3m7pc10ouq for details